firepit/sqlstorage.py

```diff
@@ -899,27 +899,19 @@ def number_observed(self, viewname, path, value=None):
         else:
             count = self.count(viewname)
         return count
-
-    def timestamped(
+        
+    def extract_observeddata_attribute(
             self,
             viewname,
+            name_of_attribute,
             path=None,
             value=None,
-            timestamp='first_observed',
             limit=None,
             run=True):
         """
-        Get the timestamped observations of `value` in `viewname`.`path`
-        Returns list of dicts like {'timestamp': '2021-10-...', '{column}': '...'}
+        Get the observations of `value` in `viewname`.`path`
+        Returns list of dicts like {'an_attribute_of_SCO': '...', '{column}': '...'}
```

Comment:
`observed-data` is an SDO, not an SCO.  The comment might be more clear if it read:
> like {`name_of_attribute`: ...}